shadowmapping/shadowmapping.cpp

```diff
@@ -526,8 +526,7 @@ class VulkanExample : public VulkanExampleBase
 	{
 		VkResult err;
 		VkSemaphore presentCompleteSemaphore;
-		VkSemaphoreCreateInfo presentCompleteSemaphoreCreateInfo =
-			vkTools::initializers::semaphoreCreateInfo(VK_FLAGS_NONE);
+		VkSemaphoreCreateInfo presentCompleteSemaphoreCreateInfo = vkTools::initializers::semaphoreCreateInfo();

```

Comment:
GH is showing this diff incorrectly, for the actual change see https://github.com/SaschaWillems/Vulkan/pull/64.patch